tests/phpunit/integration/tests/User/Preferences.php

```diff
@@ -70,12 +84,14 @@ public function tear_down() {
 	 * @covers ::can_edit_current_user
 	 */
 	public function test_add_optin_field_to_rest_api() {
+		$this->instance->register();
+
 		wp_set_current_user( self::$admin_id );
 
 		add_user_meta( self::$admin_id, \Google\Web_Stories\User\Preferences::OPTIN_META_KEY, true );
 		add_user_meta( self::$admin_id, \Google\Web_Stories\User\Preferences::MEDIA_OPTIMIZATION_META_KEY, true );
 
-		$request  = new WP_REST_Request( \WP_REST_Server::READABLE, sprintf( '/web-stories/v1/users/%d', self::$admin_id ) );
+		$request  = new WP_REST_Request( \WP_REST_Server::READABLE, sprintf( '/wp/v2/users/%d', self::$admin_id ) );
```

Comment:
Why change this?

Reply:
https://github.com/google/web-stories-wp/pull/8513#discussion_r721332879